Makefile

```diff
@@ -29,7 +29,7 @@ ifeq ($(OS),Darwin)
 install-watch-deps: ## Install MacOS dependencies for watching docs
 	brew install fswatch
 else
-install-watch-deps: ## Install Linux dependencies for watching docs
+install-watch-deps: ## Install Debian-based OS dependencies for watching docs
```

Comment:
More specific notes for developers if they are using other OS other than Debian-based